unix/qstrdefsport.h

```diff
@@ -79,6 +79,7 @@ Q(listen)
 Q(accept)
 Q(recv)
 Q(recvfrom)
+Q(settimeout)
```

Comment:
@dhylands: That makes a lot of sense. Is there a way to programmatically tell which feature needs which Q(xxx), provided that the (feature) header files that reference the header files generated from Q(xxx) are also generated programmatically as far as I can tell?

In a more concrete example, to be able to compile the Windows port using the lwIP lib I had to go down a wild goose chase for the header that was generated by X script and referenced a header generated by Y script; not a fan of code auto-generation but I guess it works. Eventually, I sort of *guessed* that the Q(settimeout) would give me the header that I needed. Is there a better approach than going down that path and adding an #ifdef for every possible lib after the problem is encountered?